### DISCUSSION TITLE
Unspent money to go to next month

### DISCUSSION THREAD

**Discussion by gregblink** [firefly-iii/firefly-iii#4473] [Firefly III] [answered]
Hiya, Is there any way to do that ? Or is this something you can consider adding in the roadmap ?

**Comment by JC5** [comment, marked as answer]
I assume you want Firefly III to automatically set your budgets to the original amount + whatever was left from the previous month? That will never be added. Since you're in control of your finances, you decide what to do with money that's left over. If you have money left, save it and use it for a piggy bank.

https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/personal-finances/

**Reply by gregblink** [reply]
Thanks 👍 :)
